src/test/PoolNestedActionsTest.sol

```diff
@@ -46,14 +46,8 @@ contract PoolNestedActionsTest is Test, ILockCallback {
     }
 
     function _nestedLock() internal {
-        address locker = manager.getLocker();
-        assertEq(locker, address(this));
```

Comment:
I think we should keep these lines - just checking instead that `manager.isLocked() == true` twice in this test (before and after the revert)